examples/integration_test.py

```diff
@@ -56,6 +56,22 @@ def error_cb (err):
     print('Error: %s' % err)
 
 
+class SchemaRegistry(object):
```

Comment:
Can we call this something else since looking at the sub-sequent code doesn't make it clear if the SchemaRegistry is a real thing or a mock just looking at the name. It also risks colliding with a future real schema registry class.

E.g., "MySchemaRegistry"